Sources/Fullscreen/FavoriteAdsList/Subviews/FavoriteAdsListTableHeader.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+protocol FavoriteAdsListTableHeaderDelegate: AnyObject {
+    func favoriteAdsListTableHeaderDidSelectSortingView(_ tableHeader: FavoriteAdsListTableHeader)
+}
+
+class FavoriteAdsListTableHeader: UIView {
+
+    // MARK: - Internal properties
+
+    weak var delegate: FavoriteAdsListTableHeaderDelegate?
+    weak var searchBarDelegate: UISearchBarDelegate? {
+        didSet { searchBar.delegate = searchBarDelegate }
+    }
+
+    internal var searchBarPlaceholder: String = "" {
```

Comment:
We can skip `internal` here and for the properties below.

Reply:
👍